application/src/main/java/org/opentripplanner/updater/trip/UrlUpdaterParameters.java

```diff
@@ -4,4 +4,8 @@ public interface UrlUpdaterParameters {
   String url();
   String configRef();
   String feedId();
+
+  default boolean producerMetrics() {
+    return false;
+  }
```

Comment:
Isn't that orthogonal to this type?

Reply:
Not sure what you're suggesting here. Can you elaborate what you mean?

Reply:
Ah, this is the base type for the various Siri ET params. I was a bit concerned that this carries the information into the GTFS code, which doesn't know anything about this feature.

Do you need this on the interface or can it just be on the concrete implementations?

Reply:
It is accessed in the StreamingTripUpdateMetrics class which only has access to the interface. I suppose I could push the evaluation of which tags to add to metrics could be pushed elsewhere, but this would be a bigger change that I'm not sure would be the right direction to go.